ztp/source-crs/SriovVrbClusterConfig.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: sriovvrb.intel.com/v1
+kind: SriovVrbClusterConfig
+metadata:
+  name: config
+  namespace: vran-acceleration-operators 
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  drainSkip: $drainSkip # true if SNO, false by default
+  priority: 1
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  acceleratorSelector:
+    pciAddress: $pciAddress
+  physicalFunction:  
+    pfDriver: "vfio-pci"
+    vfDriver: "vfio-pci"
+    vfAmount: 16
+    bbDevConfig: $bbDevConfig
```

Comment:
How many of these are things users would potentially customize?

Reply:
It is dependent on mainly on the workload that consumes these accelerated VF resources, but also CSP requirements.  Best to leave this flexible in ZTP but point to Intel references were available, IMHO.